float32 torque
float32 speed
float32 position
float32 temperature
uint16 error
